src/main/java/org/broadinstitute/dsm/util/SystemUtil.java

```diff
@@ -35,6 +37,13 @@
     private static final String LINEBREAK = "\r";
     public static final String SEPARATOR = "\t";
 
+    public static String getISO8601DateString() {
+        TimeZone tz = TimeZone.getTimeZone("UTC");
+        DateFormat df = new SimpleDateFormat(ISO_8601_UTC_DATE_STRING);
+        df.setTimeZone(tz);
+        return df.format(new Date());
+    }
+
```

Comment:
@SimoneMaiwald I would use everything related to date from `java.time` package, because it was introduced with Java 8 and is much more stable, date related classes from `java.text` are pretty old and they may have problems on new version of Java